src/lib/actions/RuleDescribeAction.ts

```diff
@@ -0,0 +1,94 @@
+import {Action} from "../ScannerCommand";
+import {Inputs, Rule} from "../../types";
+import {AnyJson} from "@salesforce/ts-types";
+import {RuleFilter} from "../RuleFilter";
+import {Controller} from "../../Controller";
+import {BundleName, getMessage} from "../../MessageCatalog";
+import {deepCopy} from "../util/Utils";
+import Dfa from "../../commands/scanner/run/dfa";
+import Run from "../../commands/scanner/run";
+import {Ux} from "@salesforce/sf-plugins-core";
+import {Display} from "../Display";
+import {RuleFilterFactory} from "../RuleFilterFactory";
+
+type DescribeStyledRule = Rule & {
+	runWith: string;
+	enabled: boolean;
+};
+
+/**
+ * The Action behind the "rule describe" command
+ */
+export class RuleDescribeAction implements Action {
+	private readonly display: Display;
+	private readonly ruleFilterFactory: RuleFilterFactory;
+
+	public constructor(display: Display, ruleFilterFactory: RuleFilterFactory) {
+		this.display = display;
+		this.ruleFilterFactory = ruleFilterFactory;
+	}
+
+	public async validateInputs(_inputs: Inputs): Promise<void> { // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
Since the method explicitly returns a Promise and doesn't use `await`, there's no need for `async`.

Reply:
ok will remove.

Reply:
Fixed - squashed into part 10 with rebase and then force push.